src/main/java/seedu/address/logic/parser/EditCommandParser.java

```diff
@@ -142,6 +143,30 @@ public EditCommand parse(String args) throws ParseException {
             }
 
             return new EditSkillCommand(index, editSkillDescriptor);
+        case ItemUtil.NOTE_ALIAS:
+            EditNoteDescriptor editNoteDescriptor = new EditNoteDescriptor();
+            boolean isAnyFieldEdited = false;
+
+            if (argMultimap.getValue(PREFIX_NAME).isPresent()) {
+                isAnyFieldEdited = true;
+                editNoteDescriptor.setName(ParserUtil.parseName(argMultimap.getValue(PREFIX_NAME).get()));
+            }
+            if (argMultimap.getValue(PREFIX_TIME).isPresent()) {
+                isAnyFieldEdited = true;
+                editNoteDescriptor.setTime(ParserUtil.parseTime(argMultimap.getValue(PREFIX_TIME).get()));
+            }
+            if (argMultimap.getValue(PREFIX_DONE).isPresent()) {
```

Comment:
Suggestion to not include the ability to have a edit 1 d/adsfkasd to set as done if there is already a done method